orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/tasks/MonitorBuildJobTask.groovy

```diff
@@ -32,7 +32,7 @@ import java.util.concurrent.TimeUnit
 
 @Slf4j
 @Component
-class MonitorJenkinsJobTask implements RetryableTask {
+class MonitorBuildJobTask implements RetryableTask {
```

Comment:
Yeah, don't rename tasks. This causes serialization problems trying to read pipelines using the old task name. Here is an example of the type of work that needs to be done afterwards.  https://github.com/spinnaker/orca/pull/1198